backend/ventas_servicio/gestor_ventas/app.py

```
@@ -19,8 +20,8 @@
 app.register_blueprint(home_bp, url_prefix=URL_HOME)
 app.register_blueprint(visita_cliente_bp, url_prefix=URL_PREFIX_PLANES)
 app.register_blueprint(ruta_visitas_bp, url_prefix=URL_PREFIX)
+app.register_blueprint(ruta_visita_optimizada_bp, url_prefix=URL_PREFIX)
 CORS(app)
 
 
-if __name__ == '__main__':
-    app.run(debug=True, host="0.0.0.0", port=3008)
\ No newline at end of file
+if __name__ == '__main__':    app.run(debug=True, host="0.0.0.0", port=3008)
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)